### PR TITLE
Qdk march rel notes

### PR DESCRIPTION
**Pull request by bromeg**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/aee4bc0660d38af6ed68fb6d26b271e052bd91ec/articles/relnotes/index.md) | 76 | 41 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/788f9a80-56f9-4030-b47c-d385f3986547_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/relnotes/index.md [hunk 1]
```diff
@@ -17,6 +17,16 @@ For installation instructions, please refer to the [install guide](xref:microsof
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
 
+## Version 0.xx.xxxx.xxxx
+
+*Release date: March 26th, 2020*
+
+This release contains the following:
+
+- New support for access modifiers in Q#, for more information see [File Structures](xref:microsoft.quantum.language.file-structure)
```
Comment (cgranade) [inline articles/relnotes/index.md]: May be good to note that we switched to the LTS version of .NET Core, such that .NET Core SDK 3.1 is now required. (Please correct me if I'm wrong, @bettinaheim!)

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/4eda848cb5bff851edb41606d169e1b4bc44bd85/articles/relnotes/index.md) | 76 | 41 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/7fbdfbaf-68a5-4bdf-8e65-6f7de33fc714_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

File: articles/relnotes/index.md [hunk 2]
```diff
@@ -17,6 +17,17 @@ For installation instructions, please refer to the [install guide](xref:microsof
 For update instructions, please refer to the [update guide](xref:microsoft.quantum.update).
 
 
+## Version 0.xx.xxxx.xxxx
```
Comment (cgranade) [inline articles/relnotes/index.md]: We now have the final release version number. Thanks for your patience!
```suggestion
## Version 0.11.2003.2506
```

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum Acrolinx score of 80 is required.**

Click the scorecard links for each article to review the Acrolinx feedback on grammar, spelling, punctuation, writing style, and terminology.

| Article | Acrolinx<br>score | Lowest<br>category<br>score | Spelling<br>issues | Scorecard | Processed |
|---------|:--------------:|:--------------------:|:------:|:---------:|:---------:|
| [articles/relnotes/index.md](https://github.com/MicrosoftDocs/quantum-docs-pr/blob/ff63869e3dbd4c24aa6872545bcfac447b8e1971/articles/relnotes/index.md) | 76 | 41 | 20 | [link](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/a52e4c04-0d3f-4697-ae71-ded905577c26_report.html) | [:white_check_mark:](# "Successfully processed") |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Pull request merged**